### DISCUSSION TITLE
Unable to start the docker container on GCP

### DISCUSSION THREAD

**Discussion by ipavlic-pointcare** [nextgenhealthcare/connect#5546] [Q&A]
I am unable to use the docker image on through GCP's Cloud Run, with an error `failed to start and then listen on the port defined by the PORT=8080 environment variable`. I tried providing the 8443 instead, which also failed. Was someone able to use the docker image to start the container on GCP?

**Comment by pacmano1** [comment]
If you post here and the forums, please cross reference the posts.

Cross post at https://forums.mirthproject.io/forum/mirth-connect/support/182629-using-the-docker-image-on-gcp-s-cloud-run#post182629

**Reply by ipavlic-pointcare** [reply]
Thank you for the heads up. I haven't seen that rule anywhere. I read the notice on the forums:

> Hi everyone, please check out our new discussion forum located in our GitHub site: https://github.com/nextgenhealthcare/connect/discussions. Let us know your feedback as we try out this new part of GitHub! Don't worry, the content in these forums won't be going away and will at a minimum be preserved for historical reference.

and posted here instead. Thanks!

**Reply by pacmano1** [reply]
Not giving you a hard time - they (nextgen) should make that a bit more clear.   As for GCP, I think very few Mirth folks use it, rather they are in AWS/Azure.  I hope someone chimes in for you though.

**Reply by ipavlic-pointcare** [reply]
Thanks! I'm happy with AWS/Azure proposals as well.

**Reply by nathan-malinowski** [reply]
is this still an issue?
